CTestConfig.cmake

```diff
@@ -13,7 +13,12 @@
 #
 set(CTEST_PROJECT_NAME "pika")
 set(CTEST_NIGHTLY_START_TIME "00:00:00 GMT")
+set(CTEST_DROP_METHOD "https")
 
-set(CTEST_SUBMIT_URL
-    "https://cdash.cscs.ch/submit.php?project=${CTEST_PROJECT_NAME}"
-)
+set(CTEST_DROP_SITE "cdash.cscs.ch")
+set(CTEST_DROP_LOCATION "/submit.php?project=pika")
+set(CTEST_DROP_SITE_CDASH TRUE)
+
+#set(CTEST_SUBMIT_URL
+#    "https://monteggio-new.cscs.ch/submit.php?project=pika"
+#)
```

Comment:
Was this for testing?

Reply:
Sorry. Thought I had deleted that. I have removed it and force pushed (yes it was for testing)

Reply:
No worries, and thanks!